mbed_host_tests/host_tests_runner/host_test_default.py

```diff
@@ -500,8 +500,13 @@ def execute(self):
             if self.options.skip_flashing:
                 self.logger.prn_inf("copy image onto target... SKIPPED!")
             else:
-                self.logger.prn_inf("copy image onto target...")
-                result = self.mbed.copy_image()
+                result = None
+                for cnt in range(1, 4):
```

Comment:
i agree with @bridadan  on different failures, should we re-try only if there is a flashing failure like fail.txt, assert.txt is generated, We can log the failure as well so we don't miss what was the reason for failure.

We get fail.txt for all mbed supported devices, adding @c1728p9 .